### DISCUSSION TITLE
Create new picklist to be used in Invoice Proforma module

### DISCUSSION THREAD

**Discussion by rafalcichon-wydajnafirmapl** [YetiForceCompany/YetiForceCRM#15555] [How to] [answered]
In invoice module I am able to create separate numbering for invoices based on invoice type. I am choosing proper picklist and that's it.
I need such solution for Invoice Proforma. 

Is it possible to add separate numbering for different types of invoice proforma?

**Comment by rskrzypczak** [comment, marked as answer]
Since you managed to do it in the Invoices module, you shouldn't have a problem with the Proforma module. There is only one mechanism, so it should work the same everywhere.

**Reply by rafalcichon-wydajnafirmapl** [reply]
Yes, but for a Sales Invoices by default is set picklist "invoice type" which can be chosen for numbering. But for Proforma Invoices field "Picklist Variables" is empty.
Proforma Invoices, screen from prefixes configuration:
![image](https://user-images.githubusercontent.com/78164458/146569395-f2ea4488-10fe-4d34-95e2-84327745ad15.png)

Sales Invoices, screen from prefixes configuration:
![image](https://user-images.githubusercontent.com/78164458/146569595-ca2cbd77-c7b8-4ef8-8722-6b68e5fe8ad0.png)

**Reply by rskrzypczak** [reply]
You need to set a prefix for the picklist value:
![image](https://user-images.githubusercontent.com/10221999/146570305-08ac0124-01e8-48b9-9771-b6e5ec7051a4.png)

As you probably have for the invoice type picklist

**Reply by rafalcichon-wydajnafirmapl** [reply]
Thank you for your quick answers!

My question, maybe wrongly stated, was about additional value in below list:
![image](https://user-images.githubusercontent.com/78164458/146571135-a531cfdd-23ce-4547-ab15-f94b1e3694d4.png)
There is a payment method and status. I need invoice type.

**Reply by rskrzypczak** [reply]
So just add a new picklist field :) (Software configuration/Standard modules/Edit fields)
https://www.youtube.com/watch?v=J14paUYtVwI

**Reply by rafalcichon-wydajnafirmapl** [reply]
Thank you so much for your help!
